CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+Deprecations:
+ - Removed support for the .NET Framework 3.5.
```

Comment:
Do we want to release a 4.4.1 with the pending fixes before we get into a 5.0? Create a branch for 5.0?

Reply:
Sounds good. Could you publish a 4.4.1 publish now (without waiting for any further PRs&mdash;I think they can all go into a 5.0.0)? Then I'll rebase this PR right afterwards.

Reply:
Will do, #498.